### PR TITLE
CSS fixes for organizations and groups

### PR DESCRIPTION
**Pull request by zelima**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+list:
+	@grep '^\.PHONY' Makefile | cut -d' ' -f2- | tr ' ' '\n'
+
+
+assets:
+	npx gulp less
+
+i18n_compile:
+	docker-compose -f ../../docker-compose.dev.yml exec ckan-dev bash -c 'cd /srv/app/src_extensions/ckanext-ed && python setup.py compile_catalog -l en -f'
```
Comment (bobeal) [inline Makefile]: It seems we don't have the referenced `docker-compose.dev.yml` ?

File: ckanext/ozwillo_theme/templates/package/search.html [hunk 2]
```diff
@@ -2,8 +2,7 @@
 
 {% block secondary_content %}
 
-  {% snippet "spatial/snippets/spatial_query.html", default_extent="{ \"type\": \"Polygon\", \"coordinates\": [[[-5.1406, 41.3337], [9.5593, 51.0891]]]}" %}
```
Comment (bobeal) [inline ckanext/ozwillo_theme/templates/package/search.html]: Does it mean we have lost the spatial query feature ?

File: create.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import requests
```
Comment (bobeal) [inline create.py]: what is it used for ?

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,41 @@
+.PHONY: assets i18n_compile i18n_extract readme pull start shell test remove stop
```
Comment (zelima) [inline Makefile]: @tanvirchahal We don't need Makefile. Let's delete

File: ckanext/ozwillo_theme/templates/package/search.html (hunk 2 above)
Comment (zelima) [inline ckanext/ozwillo_theme/templates/package/search.html]: We need this back

File: ckanext/ozwillo_theme/templates/spatial/snippets/spatial_query.html [hunk 5]
```diff
@@ -0,0 +1,29 @@
+{#
```
Comment (zelima) [inline ckanext/ozwillo_theme/templates/spatial/snippets/spatial_query.html]: This file is present in ckanext-spatial and I don't see any difference. Why do we need to copy across?  https://github.com/ckan/ckanext-spatial/blob/master/ckanext/spatial/templates/spatial/snippets/spatial_query.html

File: create.py (hunk 3 above)
Comment (zelima) [inline create.py]: @tanvirchahal This also for local development only. Let's delete

File: test.ini [hunk 6]
```diff
@@ -9,7 +9,7 @@ host = 0.0.0.0
 port = 5000
 
 [app:main]
-use = config:../ckan/test-core.ini
+use = config:../../src/ckan/test-core.ini
```
Comment (zelima) [inline test.ini]: Please checkout changes in this file as well. None of them related to PR

**Review by zelima — COMMENTED** [review]
@bobeal All of those strange changes were for local development. I did not review this properly  - only the visual part. 

@tanvirchahal left a few comments. Please make the requested changes.

**Pull request merged**